scripts/env-setup.mk

```diff
@@ -1,6 +1,9 @@
 # File: env-setup.mk
 
-CCACHE_DIR := $(CURDIR)/buildroot/output/ccache
+# Allow override of ccache dir from env var
+ifeq ($(CCACHE_DIR),)
```

Comment:
Why change the default?  The `buildroot/output/ccache` is used by the "ccache pre-population" machinery.

Reply:
The default stays the same; but for enabling a persistent ccache for the jenkins builds, I was trying a ccache dir hosted on a EFS/NFS volume. I haven't enabled it yet as I ran into issues, but this will allow overriding it as needed.
For travis builds, it should still remain the same location/mechanism.

Reply:
I'm confused as to how the default stays the same?  Doesn't this say "if it's empty, set it to `$(CURDIR)/.buildroot-ccache`?"  We used CCACHE_DIR to set BR2_CCACHE_DIR (which overrides the values in the config)-- so it seems like in the default case we'll be using `.buildroot-ccache`?

Reply:
👍